docs/_policies/leavingGSB.md

```diff
@@ -62,14 +72,18 @@ For other systems across campus, the data retention and offboarding policies may
 
 ## AWS
 
-If you use [Athena](../services/athena.html), [Cloud Forest](../services/cloudForest.html) or a custom built cloud solution, it is your responsibility to transition data off of these systems before leaving the school.
+If you've set up a AWS through Universtiy Information Technology, please go to [their cloud storage page](https://stanford.service-now.com/it_services?id=sc_category&sys_id=e6cc5a8813b9aa0019813598d144b09b){target="_blank"} to make an inquiry.
 
-{% include warning.html content="Cloud Forest is not designed for storing data, and any important results should be immediately transferred off of these ephemeral machines." %}
+If you use Athena, Cloud Forest or a custom built cloud solution, it is your responsibility to transition data off of these systems before leaving the school.
+
+!!! warning
```

Comment:
I think we can cut this warning for brevity. Nobody makes these anymore -- we can keep mention just in case, but we don't need a callout